app/views/dashboard/dashboard.html.erb

```diff
@@ -12,8 +12,8 @@
   delete ELMO.app.dashboard;
 
   ELMO.app.dashboard = new ELMO.Views.DashboardView(<%=json(
-    # needed for storing the map bounds between refreshes
-    mission_id: current_mission.id,
+    mission_id: current_mission.id, # needed for storing the map bounds between refreshes
+    reloadInterval: Rails.env.development? ? 5 : 30, # shorter interval is better for debugging
```

Comment:
This is great in general -- but the DOM gets reset every refresh which makes browser inspection more difficult. Is there an easy way to disable this in development? It's been a minor bother before but this will make it worse.

If no current method, my suggestion: support setting `window.noReload = true` via the browser console. I'm happy to submit a separate PR if you prefer.

Reply:
Yes perhaps you can do that when you next encounter it? I don't want to spend anymore time on this beast of an issue!